docs/source/pages/cli_usage/action_dump_defaults.rst

```diff
@@ -0,0 +1,25 @@
+=============
+Dump Defaults
+=============
+
+-----------
+Description
+-----------
+
+Dumps the ``defaults.ini`` file to the output directory.
+
+-----
+Usage
+-----
+
+.. code-block:: shell
+
+    usage: macaron dump-defaults [-h]
```

Comment:
This should be `./run_macaron.sh dump-defaults`.

Reply:
I wonder if we want to put the CLI Options separately from whatever the user run it with?
So in this case we only need to document the flags and action commands (analyze, verify-policy, dump-defaults) and instruct that the user could provide it directly to:
 - `macaron` if installed as a Python package
 - `./run_macaron.sh` if run as a Docker image
Or should we only give instructions on using Macaron as a Docker image for this release?

Reply:
I wonder if we want to put the CLI Options separately from whatever the user run it with?
So in this case we only need to document the flags and action commands (analyze, verify-policy, dump-defaults) and instruct that the user could provide it directly to:
 - `macaron` if installed as a Python package
 - `./run_macaron.sh` if run as a Docker image
Or should we only give instructions on using Macaron as a Docker image for this release?

Reply:
For now we assume users run Macaron as a Docker container. So let's just add `./run_macaron.sh` in the main CLI documentation for now and add `macaron` Python instructions just for contributors later.

I have created a ticket for that: https://github.com/oracle-samples/macaron/issues/272

Reply:
Addressed in https://github.com/oracle-samples/macaron/pull/269/commits/fc12961486572c6e91d62fa014813a49bf96df07